src/toil/job.py

```diff
@@ -982,9 +982,9 @@ def deleteLocalFile(self, fileStoreID):
                             os.remove(cachedFile)
                             cacheInfo.cached -= fileSize
                     self.logToMaster('Successfully deleted cached copy of file with ID '
-                                     '\'%s\'.' % fileStoreID)
+                                     '\'%s\'.' % fileStoreID, logging.DEBUG)
                 self.logToMaster('Successfully deleted local copies of file with ID '
-                                 '\'%s\'.' % fileStoreID)
+                                 '\'%s\'.' % fileStoreID, logging.DEBUG)
 
```

Comment:
I switched these from the default (logging.INFO) to DEBUG as they fill up the logging input and don't offer much helpful information from the user perspective. For my CWL runs I get hundreds of messages like:
```
Got message from job at time 07-13-2016 10:15:21: Successfully deleted local copies of file with ID 'h/y/tmpMJXEcB.tmp'.
Got message from job at time 07-13-2016 10:15:21: Successfully deleted local copies of file with ID 'j/W/tmp2cwAzR.tmp'.
Got message from job at time 07-13-2016 10:15:21: Successfully deleted local copies of file with ID 'G/2/tmpGT1jJU.tmp'.
Got message from job at time 07-13-2016 10:15:21: Successfully deleted local copies of file with ID 'a/W/tmpUWRh4C.tmp'.
Got message from job at time 07-13-2016 10:15:21: Successfully deleted local copies of file with ID 'A/l/tmpwS225a.tmp'.
Got message from job at time 07-13-2016 10:15:21: Successfully deleted local copies of file with ID 'Q/f/tmpMt3I3j.tmp'.
```
In my mind these seemed better suited for debugging than running information.